Make go doc comments compliant with `gofmt`

hsteidl opened the pull request:
Make go doc comments compliant with `gofmt` by replacing double spaces with bullet points.

This is the result of a discussion [here](https://github.com/grid-x/modbus/pull/66/files#r1158452083).

andig commented on asciiclient.go:
```diff
@@ -54,13 +54,13 @@ func (mb *asciiPackager) SetSlave(slaveID byte) {
-//  Data            : 0 up to 2x252 chars
-//  LRC             : 2 chars
-//  End             : 2 chars
+// Encode encodes PDU in an ASCII frame:
```
Would this also get reformatted if comment started with Tab instead of -? I'd also find : following the text and then tabs more familiar.

frzifus replied:
Nevertheless, it's super weird.

andig commented:
@hsteidl please take a look at https://github.com/grid-x/modbus/pull/70/files#diff-f0426cbd040b8256849c69bbcc18df4b2364fa9cf487b21114fed91d784266feR87. I don't see any problem with applying `gofmt` without adding the kinda weird lists. Why are the lists necessary?

hsteidl commented:
I'm fine with tabs at the beginning as well 👍 I updated the commit.
I removed the weird blank lines that `gofmt` automatically inserts in front of the lists, `gofmt` doesn't complain about it when I manually remove them.

@andig I think I didn't get the "Update" part of your comment but I hope this solution fits for all of us.

Please have a look again!

andig commented:
LGTM

tretmar reviewed (approved):
lgtm 👍
